spec/index.html

```diff
@@ -91,32 +102,60 @@ <h2>Introduction</h2>
     -->
   </pre>
 
+  <p>The <a href="RDF12-CONCEPTS#dfn-rdf-dataset">RDF dataset</a> represented by an N-Quads document contains
+    exactly each <a data-cite="RDF12-CONCEPTS#dfn-quad">quad</a> matching the N-Quads
+    <a href="#grammar-production-statement"><code>statement</code></a> production.
+  </p>
+
 </section>
 
 <section>
   <h2>N-Quads Language</h2>
+
+    <p>An N-Quads document is composed of a sequence
+      of <a href="#simple-triples">simple statements</a>,
+      one for each line.
+      Lines consisteng entirely of white space  (spaces <code>U+0020</code> or tabs <code>U+0009</code>)
+      and/or comments are permitted before or after each triple.</p>
+
   <section id="simple-triples">
     <h3>Simple Statements</h3>
-    <p>The simplest statement is a sequence of (subject, predicate, object) terms
-      forming an RDF triple and an optional blank node label or IRI labeling what graph
-      in a dataset the triple belongs to,
-      all are separated by whitespace and terminated by '<code>.</code>' after each statement.</p>
+
+    <p>A simple statement extends the
+      <a data-cite="RDF12-N-TRIPLES#simple-triples">definition of simple triple</a> in [[RDF12-N-TRIPLES]]
+      with an optional <a data-cite="RDF12-CONCEPTS#dfn-named-graph">named graph</a>.</p>
+
+    <p>The simplest statement is a sequence of
+      (<a data-cite="RDF12-CONCEPTS#dfn-subject">subject</a>,
+      <a data-cite="RDF12-CONCEPTS#dfn-predicate">predicate</a>,
+      <a data-cite="RDF12-CONCEPTS#dfn-object">object</a>) terms
+      forming an <a data-cite="RDF12-CONCEPTS#dfn-rdf-triple">RDF triple</a>
+      and an optional
+      <a data-cite="RDF12-CONCEPTS#dfn-graph-name">graph name</a>
+      (a <a data-cite="RDF12-CONCEPTS#dfn-blank-node-label">blank node label</a>
+      or <a data-cite="RDF12-CONCEPTS#dfn-iri">IRI</a>) labeling what
+      <a data-cite="RDF12-CONCEPTS#dfn-named-graph">named graph</a>
+      in a <a data-cite="RDF12-CONCEPTS#dfn-rdf-dataset">dataset</a> the triple belongs to,
+      all are separated by whitespace and terminated by '<code>.</code>' after each statement.
```

Comment:
```suggestion
      all are separated by white space and terminated by '<code>.</code>' after each statement.
```
there is mixed "whitespace" and "white space" through out. W3C, at least in the past, preferred "white space".

Reply:
Yeah, I've been trying to auto-correct myself.